archive/archive_api/src/apis/bags.py

```diff
@@ -19,27 +19,30 @@
 api.add_model("Source", definition=Source)
 
 
-@api.route("/<id>")
+@api.route("/<space>/<id>")
+@api.param("space", "The space!")
```

Comment:
“space” isn’t an especially helpful term here, and this description isn’t helping!

What’s wrong with “namespace”?

Reply:
It is the will of @jtweed. OBEY.